projects/storefrontapp-e2e-cypress/cypress/integration/regression/customer-ticketing/create-ticket.e2e-spec.ts

```diff
@@ -0,0 +1,140 @@
+import { viewportContext } from "../../../helpers/viewport-context";
+import { loginRegisteredUser } from "../../../helpers/cart";
+import * as customerTicketing from '../../../helpers/customer-ticketing/cutsomer-ticketing';
+import {TestTicketDetails, TestCategory} from '../../../helpers/customer-ticketing/cutsomer-ticketing';
+
+
+
+describe('ticketing', () => {
+  viewportContext(['desktop', 'mobile'], () => {
+    context('Registered User', () => {
+      before(() => {
+        cy.window().then((win) => {
+          win.sessionStorage.clear();
+        });
+      });
+      it('should be able to create ticket when filling the required form', () => {
+        const testTicketDetails: TestTicketDetails = {
+          subject: 'Entering a subject',
+          message: 'Typing a message',
+          category: TestCategory.complaint,
+        };
+
+        loginRegisteredUser();
```

Comment:
Could you please replace with the following code?
Ishrar added it to the helper file.  
I believe it is better to depend of a shared helper rather than depending on cart helper 

import { standardUser } from '../../sample-data/shared-users';
import { generateMail, randomString } from '../user';

export function loginRegisteredUser() {
  standardUser.registrationData.email = generateMail(randomString(), true);
  cy.requireLoggedIn(standardUser);
  cy.reload();
}

Reply:
Well it's already implemented in cart e2e and I believe we are to reuse functions if already implemented

Reply:
I know, but Cart is not the same as the Shared Component used by Ishrar

Reply:
But it's exactly the same implementation and I believe he also used the implementation I added in the skeleton (which I took from Cart)

Reply:
Ishrar already integrated his changes.  He uses the one I mentioned.  I don't think we should have two ways of doing the same thing.